contracts/V2/factories/CompoundingRewardsPoolFactory.sol

```diff
@@ -12,12 +12,13 @@ contract CompoundingRewardsPoolFactory is AbstractPoolsFactory, StakeTransferEna
 	using SafeERC20Detailed for IERC20Detailed;
 
 	address public immutable externalRewardToken;
+	uint256 public constant MAX_INT = 115792089237316195423570985008687907853269984665640564039457584007913129639935;
```

Comment:
I think it is normally written as unit(-1), but that is good too

Reply:
uint256 MAX_INT = 2**256 - 1; it can be written like this also, but I was thinking this was intended

Reply:
but I think the more easy way to write this is: `type(uint256).max;`, but I don't know if is from the 0.7 or the 0.6 if this is available